proto/process/agent.proto

```diff
@@ -620,6 +620,9 @@ message Connection {
 
 	// serialized HTTPAggregations object summarizing all http transactions recorded for this connection, organized by request path
 	bytes httpAggregations = 37;
+
+	// The stored keys corresponds to an index into the `classificationTags` field
+	repeated int32 classification = 43;
```

Comment:
I don't think we need to scope these to just classification. Perhaps name this something simple like `tags`.

Reply:
done